lib/DDG/Spice/Maven.pm

```diff
@@ -4,7 +4,7 @@ package DDG::Spice::Maven;
 use strict;
 use DDG::Spice;
 
-triggers startend => "maven", "mvn";
+triggers startend => "maven", "mvn","library","repository","repo","package";
```

Comment:
These triggers are a bit too general - it means that a search for `library book` would trigger this IA.

Instead, please could they be changed to `"maven library", "maven repository", "maven repo", "maven package"`? That way, only maven-related searches will trigger this IA.